turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -2966,8 +3009,9 @@ impl VisitAstPath for ModuleReferencesVisitor<'_> {
         export: &'ast ExportAll,
         ast_path: &mut AstNodePath<AstParentNodeRef<'r>>,
     ) {
-        let path = Vc::cell(as_parent_path(ast_path));
-        self.analysis.add_code_gen(EsmModuleItem::new(path));
+        let path = ResolvedVc::cell(as_parent_path(ast_path));
+        self.analysis
+            .add_code_gen(EsmModuleItem { path }.resolved_cell());
```

Comment:
That changes the semantic, but I see why you did this. This could also be addressed by resolving all code_gens at the end of this turbo-tasks function